packages/documentation/pages/usage/components/field-inline-edit.vue

```diff
@@ -0,0 +1,152 @@
+<template>
```

Comment:
Why is this not in the form fields page?

Reply:
Let's not block the MR for this.
Merging documentations can be a follow-up.